overlore/townhall/service.py

```diff
@@ -48,27 +47,38 @@ async def start() -> None:
     if TORII_WS is None:
         raise RuntimeError("Failure to provide WS url")
 
-    db = DatabaseHandler.instance().init()
+    DatabaseHandler.instance().init()
 
     signal.signal(signal.SIGINT, handle_sigint)
     gpt_interface = GptInterface.instance()
     gpt_interface.init(OPENAI_API_KEY, OPENAI_EMBEDDINGS_API_KEY)
 
     bound_handler = functools.partial(service, extra_argument={"mock": args.mock})
-    overlore_pulse = serve(bound_handler, args.address, SERVICE_WS_PORT)
+    serve(bound_handler, args.address, SERVICE_WS_PORT)
 
     print(f"great job, starting this service on port {SERVICE_WS_PORT}. everything is perfect from now on.")
-    # arbitrarily choose just the first realm to sub to events on
-    await asyncio.gather(
-        overlore_pulse,
-        torii_event_sub(TORII_WS, db.process_event, Subscriptions.COMBAT_OUTCOME_EVENT_EMITTED),
-        torii_event_sub(TORII_WS, db.process_event, Subscriptions.ORDER_ACCEPTED_EVENT_EMITTED),
-    )
+
+    conn = sqlean.connect(args.world_db)
+
+    print("sanity")
+
+    while True:
```

Comment:
there was an issue with reliably establishing web socket connections, so I commented them out. then with those commented out, the overlore_pulse service wouldn't block on the gather call, so the program would just terminate on start.

that's why, temporarily, i commented it out and just put a while true loop and queried the world.db that the litefs service is meant to sync